nodejs/json-rpc-relay/integration.tests/mirror.node/get.hashrate.test.js

```diff
@@ -0,0 +1,55 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const Web3 = require('web3');
+const { curly } = require('node-libcurl')
+
+const httpHeaderJson = [
+    'Content-Type: application/json',
+    'Accept: application/json'
+  ];
+const arkhiaJsonRpcRelayTestnet = `${process.env.ARKHIA_TESTNET_URL}/${process.env.ARKHIA_API_KEY}`;
+const communityHashioMainnet = process.env.COMMUNITY_MAINNET_URL;
+
+const getPayload = () => {
+    const data = {
+        "jsonrpc": "2.0",
+        "method": "eth_hashrate",
+        "params": [],
+        "id": 0
+      };
+    return data;
+}
+
+describe('[CURL] Hashrate', () => {
+
+    test('Should get 0x0 from Arhia Testnet', async () => {
```

Comment:
Curious have we tried to cross check with Hashio? JUst wanna confirm 0x0 is actually the correct hashrate